### ISSUE TITLE
Mac touchpad issue

### ISSUE THREAD

**Issue by Parthjasani7** [lukehaas/Scrollify#195]
hey!
I don't know if this issues is reported yet:

If i use my trackpad or the magicmouse on an page with the scrollify.js script, the scrolling event is triggered twice. it scrolls down and skips the third page. help?

**Comment by lukehaas** [comment]
I've observed this issue when the `scrollSpeed` is set to a low number. You could try tweaking that to see if it makes a difference.

**Comment by Florrpan** [comment]
I can confirm, using laptop (windows) touchpad. When scrolling, or "swiping" the fingers up and down, the scroll wants to continue scrolling after the snapping have taken place, which causes issues because it continues to the next section. It seems to happen frequently on sections that requires overflow scrolling (content longer than the screen)

**Comment by dominicfallows** [comment]
I can also confirm on Mac, the problem is perhaps with the inertia scroll effects added to most OS/Browsers now. Multiple (many multiple on MagicMouse/Mac Trackpad) extra mousewheel events are triggered. I wonder whether using timeouts when receiving mousewheel events is a solution? (therefore ignoring the extra events, only actioning the first).

**Comment by lukehaas** [comment]
Yes, fully aware of the issue and it's cause. The only way to deal with kinetic scrolling is to work out if the scroll action is accelerating or decelerating. Scrollify has a function called `isAccelerating` that calculates this. Unfortunately there is no perfect solution, and the current solution only reduces the likelihood of the occurrence. Timeouts wouldn't work because kinetic scrolling produces a scroll echo that can keep on repeating for a long period of time after the user has scrolled.
